django/db/models/fields/__init__.py

```diff
@@ -843,18 +866,12 @@ def get_flatchoices(self, include_blank=True,
         first_choice = blank_choice if include_blank else []
         return first_choice + list(self.flatchoices)
 
-    def _get_val_from_obj(self, obj):
```

Comment:
I can add it back in with a deprecation warning, but it is a completely useless method. Defaults for values are already provided by model.__init__ and it duplicates the functionality of `value_from_object` (and `getattr` for that matter). I'll make a note that the documentation will need to be changed.